source/tyr/tyr/resources.py

```diff
@@ -1113,3 +1116,25 @@ def delete(self, name=None):
             raise
         return ({}, 204)
 
+class Data(flask_restful.Resource):
+    def post(self, instance_name):
+        instance = models.Instance.query_existing().filter_by(name=instance_name).first_or_404()
+        if instance is None:
+            return {'message': 'bad instance {}'.format(instance_name)}, 404
+
+        if not request.files:
+            return {'message': 'the Data file is missing'}, 400
+        content = request.files['file']
+        logger = logging.getLogger(__name__)
+        logger.info('content received: %s', content)
+
+        instance = load_instance_config('fr')
```

Comment:
The `fr` should be `instance.name`. 
Their is not real advantage to load the configuration only one time at startup, reading a file is way faster than sending a few hundreds Mo over the network
You reuse the variable instance for storing something else, I don't like that: it's error prone.